src/components/SearchPage/SearchPage.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { useState } from "react";
+
+import SearchHeader from "../SearchHeader/SearchHeader";
+import PokemonData from "../PokemonData/PokemonData";
+import Footer from "../Footer/Footer";
+import { fetchPokemon } from "../../utils/FetchPokemon";
+
+function InfoPage() {
+  const [pokeInfo, setPokeInfo] = useState();
+  const [loading, setLoading] = useState(false);
+
+  const getPokemon = async (query) => {
+    try {
+      setLoading(true);
+      const response = await fetchPokemon(query);
+      const results = await response.json();
+      console.log(results);
+      setPokeInfo(results);
+      setLoading(false);
```

Comment:
`setLoading(false);` should be in `finally` to stop the loading effect in case of an error as well


So, the final code will be like that 

```js
  const getPokemon = async (query) => {
    try {
      setLoading(true);
      const response = await fetchPokemon(query);
      if (!response.ok) {
        throw new Error(`Error ${response.status}`);
      }
      const results = await response.json();
      setPokeInfo(results);
    } catch (err) {
      console.error(err);
    } finally {
      setLoading(false);
    }
  };

```